### ISSUE TITLE
New feature: ability to publish to both web and safe

### ISSUE THREAD

**Issue by happybeing** [badcodeltd/safecms#14]
> ability to publish to both web and safe (eg via an ftp upload - I have that with my hosting service and I could provide some free accounts if others want them)

More here: https://safenetforum.org/t/safe-cms-censorship-is-a-thing-of-the-past/20419/120?u=happybeing

**Comment by BadCodeShane** [comment]
Agreed, this would be a fantastic option to add - I've added the 2.0.0 label, I'll begin working on this once the issue backlog is cleared.
